client/src/modules/depots/depots.js

```diff
@@ -12,9 +12,8 @@ DepotManagementController.$inject = [
  * This controller is about the depot management module in the admin zone
  * It's responsible for creating, editing and updating a depot
  */
-function DepotManagementController(Depots, ModalService,
-  Notify, uiGridConstants, $state) {
-  var vm = this;
+function DepotManagementController(Depots, ModalService, Location, Notify, uiGridConstants, $state) {
```

Comment:
It would be better to call this `Locations`, since that has been our practice with most other services (for example, `Depots`), and `location` is a reserved JS word that may confuse future developers.  GitHub's syntax highlighting certainly thinks so!